hack/make/binary-daemon

```diff
@@ -14,7 +14,7 @@ copy_binaries() {
 		return
 	fi
 	echo "Copying nested executables into $dir"
-	for file in containerd containerd-shim ctr runc docker-init docker-proxy rootlesskit dockerd-rootless.sh; do
+	for file in containerd containerd-shim ctr runc docker-init docker-proxy rootlesskit rootlessctl rootlesskit-docker-proxy dockerd-rootless.sh; do
```

Comment:
q: could we avoid another binary for `rootlessctl` (for size) . Eg. could it be just a subcommand of rootlesskit?

Reply:
removed rootlessctl